packages/app-desktop/gui/styles/index.scss

```
@@ -3,6 +3,7 @@
 @use './base-button.scss';
 @use './dialog-modal-layer.scss';
 @use './user-webview-dialog.scss';
+@use './note-property-box.scss';
```

Stylelint (17.4.0)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)